plugins/org.jkiss.dbeaver.ext.tidb/src/org/jkiss/dbeaver/ext/tidb/mysql/model/TiDBMySQLView.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.tidb.mysql.model;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.mysql.model.MySQLCatalog;
+import org.jkiss.dbeaver.ext.mysql.model.MySQLTableColumn;
+import org.jkiss.dbeaver.ext.mysql.model.MySQLView;
+import org.jkiss.dbeaver.model.DBUtils;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * TiDBView
+ */
+public class TiDBMySQLView extends MySQLView {
+
+    TiDBMySQLView(MySQLCatalog catalog, ResultSet dbResult) {
+        super(catalog, dbResult);
+    }
+
+    @Override
+    public List<MySQLTableColumn> getAttributes(@NotNull DBRProgressMonitor monitor) throws DBException {
+        List<MySQLTableColumn> childColumns = getContainer().getTiDBTableCache().getChildren(monitor,
+                getContainer(), this);
+        if (childColumns == null) {
+            return Collections.emptyList();
```

Comment:
I think - this is a very strange situation.
I tried to do 
`create view testView as select 1`
but testView has one column in this situation.
I think we can add logging here.
`log.error("Can't find attributes for the view " + getName())`